whitepaper/chapters/accounts.tex

```diff
@@ -4,7 +4,7 @@ \section{Accounts and Addresses}
 \nemquote{
 }{}
 
-\codenamechapterfirstword uses elliptic curve cryptography to ensure confidentiality, authenticity and non-repudiability of all transactions.
+\codenamechapterfirstword uses elliptic curve cryptography to ensure confidentiality, authenticity, and non-repudiability of all transactions.
```

Comment:
generally we've been avoiding Oxford commas, so not adding this one would be more consistent with rest of document